packages/forma-36-react-components/src/components/Table/TableBody/TableBody.tsx

```diff
@@ -1,9 +1,14 @@
 import React, { Component } from 'react';
 
+import SkeletonTable from '../../Skeleton/SkeletonTable';
+
 export type TableBodyProps = {
   className?: string;
   style?: React.CSSProperties;
   testId?: string;
+  loading?: boolean;
+  numberOfRows?: number;
```

Comment:
```suggestion
  loadingRowCount?: number;
```

Reply:
Did you consider just sticking to a single row for this?

Reply:
I started developing with a single row. But then I just experimented setting the number of rows to the limit of the pagination 

I think it's easier to explain with a real case.
Below I have a table with several users, and the pagination limit is 10.
With the loading state having one row and the final table having 10,  the table will jump from 1 to 10 rows very quickly

start with a single row...
![Screenshot 2020-03-17 at 09 58 22](https://user-images.githubusercontent.com/6597467/76839669-33837f80-6836-11ea-91c0-a27b78d5f8b2.png)

... ends with 10 rows
![Screenshot 2020-03-17 at 09 55 47](https://user-images.githubusercontent.com/6597467/76839561-0040f080-6836-11ea-8cb7-5ca15ab669a1.png)

But this is also true in the reverse case, anyways. The good thing about one row is that this prop becomes unnecessary and this means less code which means fewer bugs

I think I will change back to one then. Also, if someone needs more rows, it's possible to do it by passing more rows as children